charts/portworx/templates/hooks/pre-upgrade/retain-daemonset-install.yaml

```diff
@@ -42,8 +35,7 @@ spec:
           command: ['/bin/sh',
                     '-c',
                     'kubectl -n kube-system annotate DaemonSet portworx-api helm.sh/resource-policy=keep --overwrite;
-                     kubectl -n kube-system annotate DaemonSet portworx helm.sh/resource-policy=keep --overwrite;
-
+                     kubectl -n kube-system annotate DaemonSet portworx helm.sh/resource-policy=keep --overwrite || true;
```

Comment:
as this is preupgrade hook portworx-api exists in both daemonset and operator install, if there is other issues like api server error it would be retried.